tests/wrappers/test_multioutput.py

```diff
@@ -0,0 +1,143 @@
+from collections import namedtuple
+from functools import partial
+from typing import Any, Callable, Optional
+
+import pytest
+import torch
+from sklearn.metrics import accuracy_score
+from sklearn.metrics import r2_score as sk_r2score
+
+from tests.helpers import seed_all
+from tests.helpers.testers import BATCH_SIZE, NUM_BATCHES, MetricTester
+from torchmetrics import Metric
+from torchmetrics.classification import Accuracy
+from torchmetrics.regression import R2Score
+from torchmetrics.wrappers.multioutput import MultioutputWrapper
+
+seed_all(42)
+
+
+class _MultioutputMetric(Metric):
+    """Multi-output version of the R2 score class for testing."""
```

Comment:
This should say "Test class that allows passing base metric as a class rather than its instantiation to the wrapper."